packages/teams-js/prepBetaRelease.js

```diff
@@ -84,6 +84,10 @@ function getNewerPrefix(currBetaVer, currPkgJsonVer) {
       return currPkgPrefix;
     }
   }
+  //If we complete the loop and haven't returned, then their is an error in prefix comparisons
+  throw new Error(
+    `Error in compaison between Current Beta Version ${currBetaVer} with prefix ${currBetaPrefix} and Current Package Version ${currPkgJsonVer} with prefix ${currPkgPrefix}`,
```

Comment:
should be 'comparison'

Reply:
I knew I made a typo somewhere while I was typing this, good catch